HARK/distribution.py

```diff
@@ -1181,7 +1181,7 @@ def func_wrapper(x: np.ndarray, *args: Any) -> np.ndarray:
 
         if len(kwargs):
             f_query = func(self.dataset, **kwargs)
-            ldd = DiscreteDistributionLabeled.from_dataset(f_query, self.pmv)
+            ldd = DiscreteDistributionLabeled.from_dataset(f_query, self.probability)
```

Comment:
should probably standardize this with `DiscreteDistribution` ... that's on me

Reply:
Yeah this was the only kind of "bug" this addresses; it makes some cases work that didn't before.

The other stuff is a design question. Happy to merge this in without resolving that, but also happy to have a conversation, agree, and implement it.